rasa/core/policies/keras_policy.py

```diff
@@ -14,15 +14,17 @@
     MaxHistoryTrackerFeaturizer,
     BinarySingleStateFeaturizer,
 )
+from rasa.constants import DOCS_URL_POLICIES
 from rasa.core.featurizers import TrackerFeaturizer
 from rasa.core.policies.policy import Policy
 from rasa.core.trackers import DialogueStateTracker
-from rasa.utils.common import obtain_verbosity
+from rasa.utils.common import obtain_verbosity, raise_warning
 from rasa.core.constants import DEFAULT_POLICY_PRIORITY
 
 # there are a number of issues with imports from tensorflow. hence the deactivation
 # pytype: disable=import-error
```

Comment:
are these still required? Can we do this just for the imports and not for the entire file?

Reply:
yep, still required